### DISCUSSION TITLE
Attacks and Security Analysis on Zigbee

### DISCUSSION THREAD

**Discussion by MattWestb** [zigpy/zigpy#1149] [General]
One very deep paper is sent to CSA.-ITO and is very interesting !!
https://dl.acm.org/doi/pdf/10.1145/3548606.3560703

Silabs have verified and tested some of the attacks on EZSP and some is not working on EZSP 7.2.0 and some is unlikely but fixes is very likely coming very soon.

More i not allowed to writing then i have one license that is blocking my to do it.

**Comment by puddly** [comment]
Interesting. One of these "attacks" seems very similar to the device rejoin issue fixed in Gecko SDK 4.1.4 last week...
 <img width="279" alt="image" src="https://user-images.githubusercontent.com/32534428/214965564-5e45ab98-1d7d-40bf-8cc6-8a3c9a7f245d.png">

**Reply by MattWestb** [reply]
I think they was doing some hot fixes for GSDK 4.1.4 then the CSA-ITO was getting the papers in the summer and 8 from 10 device manufactures was replaying.

If you have one WSDK with licence you can looking on the Silabs file `A-00000447`.

They was fixing more strange bugs in that release and i think 4.2.X is getting more attention then it shall being the current stable production version. So very likely one update in some weeks after one fast beta test (4.1.4 was having one not released pre-release).

And i think then fixing some of the attacks it making paring and rejoining better but it can also making problems with bad device that is corrupting frames like OSRAM plugs can stop working then the coordinator is throwing the frame.
